app/src/main/java/com/example/maintenancespace/controllers/UserController.java

```diff
@@ -0,0 +1,12 @@
+package com.example.maintenancespace.controllers;
+
+public class UserController
+{
+
+    public static int fetchById (int userId){
+        return userId;
+    }
+    public static String fetchByEmail(String email){
+        return email;
```

Comment:
Once we implement Firebase both of these methods will be returning a user instance.  Instead of just returning the passed in email/userId can you instantiate a new user object and return that.  That way if we need to use it before Firebase is completely setup we still have the correct data being returned for testing.